src/fields/components/InternalField.tsx

```diff
@@ -74,10 +77,45 @@ export const InternalField = ({
           {errorMsg}
         </Text>
       )}
+
+      {showCompletedStatus !== undefined && (
```

Comment:
is there any specific need for checking undefined? cause this will render the whole thing also if `false` while I suppose we just need it in case it's true?

Reply:
So having spoken to Luke, out of all options- he prefers that there is white space underneath each input. 

So for `showCompletedStatus`
`undefined` => no whitespace
`false` => whitespace
`true` => complete status

Reply:
I did play with other options and I think it is possible to have it extend in height and width but then Luke still preferred whitespace. 

Is your main idea of using scale so that it does extend in height or something else?

Reply:
Yeah so the problem is that if you animate the `height` property to zero the margin top of the wrapper will still be rendered and that's what's gonna add the white space.
If you use `scale(0)` it will also include the margin so you end up having a `0 x 0` size element

Reply:
I see - I didn't know that. We can still chat with Luke if you think having it animated on y-axis would be better than whitesapce

Reply:
I'm not sure if there's another option of maybe moving the margin top to a wrapper inside the one we are animating, maybe worth a try?
So you essentially use `StatusWrapper` just for handling the display animation and then you wrap what's inside with a `Box` that handled the margin

Reply:
@leo, I just chatted to Luke and we tested the success state locally. He is happy the way it looks.

Just to sum up this conversation about the height. I understood that you are mentioning scale only in regards to animating it vertically. However, I see that what you mean is that when we have an error, despite the status having 0 height, there is still 8px margin applied to it.

So just to summarise
- we don't want margin (from status) when there is an error

Opt 1 - We set margin based on error 
```
{completed !== undefined && (
        <StatusWrapper
          displayStatus={completed}
          isError={!!(error && errorMsg)}
          mt={error && errorMsg ? '0' : '8px'} //HERE
        >
          <Icon render="included" size={16} color="success" />
          <Text typo="caption" color="success">
            Complete
          </Text>
        </StatusWrapper>
      )}
```

Opt 2 - We wrap the contents in a Box that sets margin
- AnimationWrapper does the animating
- StatusWrapper sets

```
{completed !== undefined && (
        <AnimationWrapper
          displayStatus={completed}
          isError={!!(error && errorMsg)}
        >
          <StatusWrapper mt={'8px'}>
              <Icon render="included" size={16} color="success" />
              <Text typo="caption" color="success">
                Complete
              </Text>
          </StatusWrapper>
        </AnimationWrapper>
      )}
```

```
const AnimationWrapper = styled(Box)<{
  displayStatus: boolean
  isError: boolean
}>`
  width: 0;
  overflow: hidden;

  ${({ displayStatus }) =>
    displayStatus &&
    css`
      transition: width 0.6s ease-in;
      width: 100%;
    `}

  /* This enables animation to appear when previous state is error */
  ${({ isError }) =>
    isError &&
    css`
      height: 0px;
    `}
`

const StatusWrapper = styled(Box)`
  display: flex;
  align-items: center;
  gap: 4px;
`
```

Which one do you prefer? I would actually lean towards the Opt 2. 

Btw `transform: scale(0)` [does not work for margin](https://stackoverflow.com/questions/48682497/why-does-the-scale-css-transformation-function-not-affect-margin).

Reply:
yeah I think we can go with option 2